src/proxy-ui-api/src/main/resources/common-application.yml

```diff
@@ -70,7 +70,7 @@ file-upload-endpoints:
       pathEnding: /tls-certificates
     -
       httpMethod: POST
-      pathEnding: /certificates
+      pathEnding: /token-certificates
```

Comment:
This change seems unrelated to logging, intentional?

Reply:
Yes it is intentional, I noticed that endpoint names had been updated `/certificates` -> `/token-certificates`, but this configuration file had not been updated accordingly. It is unrelated to logging, indeed.